servers/src/test/java/tachyon/worker/block/BlockDataManagerTest.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.worker.block;
+
+
+import java.io.IOException;
+import java.util.LinkedList;
+import java.util.Random;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import tachyon.Constants;
+import tachyon.Sessions;
+import tachyon.client.BlockMasterClient;
+import tachyon.client.FileSystemMasterClient;
+import tachyon.conf.TachyonConf;
+import tachyon.test.Tester;
+import tachyon.thrift.FileInfo;
+import tachyon.underfs.UnderFileSystem;
+import tachyon.util.io.PathUtils;
+import tachyon.worker.WorkerSource;
+import tachyon.worker.block.meta.BlockMeta;
+import tachyon.worker.block.meta.StorageDir;
+import tachyon.worker.block.meta.TempBlockMeta;
+
+public class BlockDataManagerTest {
+  class BlockDataManagerTester implements Tester<BlockDataManager> {
```

Comment:
What do you think make lockDataManagerTest implement Tester, and call receiveAccess in before()?